### PR TITLE
Jupyrest connector certification

### PR DESCRIPTION
**Pull request by litiany23**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Jupyrest/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,294 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "x-ms-api-annotation": {
+      "status": "Production"
```
Comment (sriyen-msft) [inline certified-connectors/Jupyrest/apiDefinition.swagger.json:5]: @litiany23 please update it to `"Preview"` as all the new connectors should be deployed with the Preview tag.  In order to update it to the "Production" tag, we require your connector to meet the following criteria after publishing it to all the production regions:

1. Connector has 99.95% reliability (calculated by % of 5xx (-502 & -504) errors) 
2. Connector has 99.5% SLO (percentage of connections meeting SLA bar above)
3. Connector has greater than 80% success rate (Calculated by % of 4xx and 5xxx errors) 
4. Connector has at least 500 weekly active users over the last 3 weeks.
5. Backend service APIs, that the connector is calling, needs to be in PRODUCTION.
6. Partner should be able provide 24/7 support for high severity incidents.

File: certified-connectors/Jupyrest/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/NotebookResponse"
+            },
+            "headers": {}
```
Comment (sriyen-msft) [inline certified-connectors/Jupyrest/apiDefinition.swagger.json:197]: @litiany23 Empty property values can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Jupyrest/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+        }
+      }
+    }
+  },
+  "parameters": {},
+  "responses": {},
+  "tags": [
+    {
+      "name": "JupyrestConnector",
+      "description": ""
```
Comment (sriyen-msft) [inline certified-connectors/Jupyrest/apiDefinition.swagger.json:271]: @litiany23 Empty property values can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Jupyrest/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      {
+        "templateId": "dynamichosturl",
+        "title": "jupyrestinstance",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "https://@connectionParameters('jupyrestinstance')/api"
+        }
+      }
+    ],
+    "publisher": "Microsoft",
+	"stackOwner": "Microsoft"
```
Comment (sriyen-msft) [inline certified-connectors/Jupyrest/apiProperties.json:28]: @litiany23 Please remove extra spaces.

**Review by sriyen-msft — DISMISSED** [review]

File: certified-connectors/Jupyrest/apiDefinition.swagger.json (hunk 1 above)
Comment (litiany23) [reply certified-connectors/Jupyrest/apiDefinition.swagger.json]: Updated it to Preview. Thanks

File: certified-connectors/Jupyrest/apiProperties.json (hunk 4 above)
Comment (litiany23) [reply certified-connectors/Jupyrest/apiProperties.json]: done

File: certified-connectors/Jupyrest/apiDefinition.swagger.json (hunk 2 above)
Comment (litiany23) [reply certified-connectors/Jupyrest/apiDefinition.swagger.json]: done

File: certified-connectors/Jupyrest/apiDefinition.swagger.json (hunk 3 above)
Comment (litiany23) [reply certified-connectors/Jupyrest/apiDefinition.swagger.json]: done

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @litiany23,

I hope you are doing well.

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. 

Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.

Please let me know if you have any questions.

Thank you very much for working with us.

**Pull request merged**